### ISSUE TITLE
Using Third-Party Libraries in Plugins

### ISSUE THREAD

**Issue by cduncan1206** [nasa/openmct#2522]
tldr; Having a hard time requiring a third-party library to use in a telemetry plugin. Sets off string of issues when changing the format of my require statements.

Hey all,

So I'm new to using the OpenMCT software, as well as javascript as a whole, and I've run into a little bit of a problem. 

I'm trying to utilize the ZeroMQ javascript bindings to write my own live telemetry plugin. I'm simply using their basic subscriber model and wrapping it into a plugin to get it to run with the application. Unfortunately, I'm running into some issues with how I'm importing, or "requiring," the zeromq library.

The example plugin that I'm trying to implement is
`function SubPlugin() {
	return function install (openmct) {
		var zmq = require('zeromq'), sock = zmq.socket('sub');

		sock.bindSync('tcp://*:4350');
		sock.subscribe('kitty cats');
		console.log('Subscriber connected to port 3000');

		sock.on('message', function(topic, message) {
		  console.log('received a message related to:', topic, 'containing message:', message);
		});
	}
}`

and I have loaded it into my OpenMCT index.html with
`<script src="node_modules/zeromq/lib/index.js></script>`

When I try to run the application with all of this, the application never passes the load screen, and I get the following string of errors:

require.js:165 Uncaught Error: Module name "events" has not been loaded yet for context: _. Use require([])
http://requirejs.org/docs/errors.html#notloaded

require.js:900 Error: Module name "zeromq" has not been loaded yet for context: _. Use require([])
http://requirejs.org/docs/errors.html#notloaded
    

When I then try to restructure my require functions to use the method utilizing the brackets, as they mention, it sets off a string of similar errors, from zeromq to events/eventemitter to d to es5-ext/object/assign, asking me to rewrite the requires to utilize that method, even when impossible to do so.

Does anyone have any experience with this? Maybe not specifically with ZeroMQ, but with loading a third-party library?

Thank you!
